compiler/noirc_evaluator/src/ssa/ir/printer.rs

```diff
@@ -172,5 +172,8 @@ pub(crate) fn display_instruction(
                 show(*value)
             )
         }
+        Instruction::RangeCheck { value, max_bit_size, .. } => {
+            write!(f, "range {} to {} bits", show(*value), *max_bit_size,)
```

Comment:
```suggestion
            write!(f, "range_check {} to {} bits", show(*value), *max_bit_size,)
```
Having just "range x to y bits" reads a bit awkward to me.

Reply:
Done